<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2042</width>
    <height>1158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FACET-II CUD: Long. FB History</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 10);
color: rgb(255, 255, 255);

PyDMLabel {
  border-radius: 5px;
}
PyDMLabel[alarmSensitiveBorder=&quot;true&quot;][alarmSeverity=&quot;0&quot;] {
  background-color: rgb(0,255,0);
  color: rgb(0,0,0);
}

PyDMLabel[alarmSensitiveBorder=&quot;true&quot;][alarmSeverity=&quot;1&quot;] {
  background-color: rgb(255,255,0);
  color: rgb(0,0,0);
}

PyDMLabel[alarmSensitiveBorder=&quot;true&quot;][alarmSeverity=&quot;2&quot;] {
  background-color: rgb(255,0,0);
  color: rgb(255,255,255);
}

PyDMLabel[alarmSensitiveBorder=&quot;true&quot;][alarmSeverity=&quot;3&quot;] {
  background-color: rgb(192,0,255);
  color: rgb(255,255,0);
}

PyDMLabel[alarmSensitiveBorder=&quot;true&quot;][alarmSeverity=&quot;4&quot;] {
  color: rgb(218,218,218);
  background-color: rgb(25,25,25);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,1,1,1,1,1">
      <item row="6" column="1">
       <widget class="PyDMTimePlot" name="L3_blen">
        <property name="toolTip">
         <string/>
        </property>
        <property name="showXGrid">
         <bool>false</bool>
        </property>
        <property name="showYGrid">
         <bool>false</bool>
        </property>
        <property name="yAxes">
         <stringlist>
          <string>{&quot;name&quot;: &quot;Axis 1&quot;, &quot;orientation&quot;: &quot;right&quot;, &quot;label&quot;: null, &quot;minRange&quot;: -189.7978527744, &quot;maxRange&quot;: 189.7978527744, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
          <string>{&quot;name&quot;: &quot;Axis2&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: &quot;  &quot;, &quot;minRange&quot;: -339.79785277440004, &quot;maxRange&quot;: 39.7978527744, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
         </stringlist>
        </property>
        <property name="curves">
         <stringlist>
          <string>{&quot;channel&quot;: &quot;BLEN:LI20:3014:BZ203014B_S_SUM&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;BLEN:LI20:3014:BZ203014B_S_SUM&quot;, &quot;color&quot;: &quot;#ffff7f&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 1&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
         </stringlist>
        </property>
        <property name="updateMode" stdset="0">
         <enum>PyDMTimePlot::AtFixedRate</enum>
        </property>
        <property name="timeSpan">
         <double>900.000000000000000</double>
        </property>
        <property name="autoRangeY">
         <bool>true</bool>
        </property>
        <property name="minYRange">
         <double>0.000000000000000</double>
        </property>
        <property name="maxYRange">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="PyDMTimePlot" name="L0_energy">
        <property name="toolTip">
         <string/>
        </property>
        <property name="showXGrid">
         <bool>false</bool>
        </property>
        <property name="showYGrid">
         <bool>false</bool>
        </property>
        <property name="yAxes">
         <stringlist>
          <string>{&quot;name&quot;: &quot;Axis 1&quot;, &quot;orientation&quot;: &quot;right&quot;, &quot;label&quot;: null, &quot;minRange&quot;: -5693.935583232, &quot;maxRange&quot;: 5693.935583232, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
          <string>{&quot;name&quot;: &quot;Axis 2&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: null, &quot;minRange&quot;: -5693.935583232, &quot;maxRange&quot;: 5693.935583232, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
         </stringlist>
        </property>
        <property name="curves">
         <stringlist>
          <string>{&quot;channel&quot;: &quot;BPMS:IN10:731:X1H&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;BPMS:IN10:731:X1H&quot;, &quot;color&quot;: &quot;#00aaff&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 1&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;KLYS:LI10:41:ADES&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;KLYS:LI10:41:SFB_ADES&quot;, &quot;color&quot;: &quot;#ffaaff&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 2&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
         </stringlist>
        </property>
        <property name="updateMode" stdset="0">
         <enum>PyDMTimePlot::AtFixedRate</enum>
        </property>
        <property name="timeSpan">
         <double>900.000000000000000</double>
        </property>
        <property name="autoRangeY">
         <bool>true</bool>
        </property>
        <property name="maxYRange">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="PyDMTimePlot" name="L1_energy">
        <property name="toolTip">
         <string/>
        </property>
        <property name="showXGrid">
         <bool>false</bool>
        </property>
        <property name="showYGrid">
         <bool>false</bool>
        </property>
        <property name="yAxes">
         <stringlist>
          <string>{&quot;name&quot;: &quot;Axis 1&quot;, &quot;orientation&quot;: &quot;right&quot;, &quot;label&quot;: null, &quot;minRange&quot;: -5474.9380608, &quot;maxRange&quot;: 5474.9380608, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
          <string>{&quot;name&quot;: &quot;Axis 2&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: null, &quot;minRange&quot;: -5474.9380608, &quot;maxRange&quot;: 5474.9380608, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
         </stringlist>
        </property>
        <property name="curves">
         <stringlist>
          <string>{&quot;channel&quot;: &quot;BPMS:LI11:333:X1H&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;BPMS:LI11:333:X1H&quot;, &quot;color&quot;: &quot;#00aaff&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 1&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;KLYS:LI11:11:AREQ&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;KLYS:LI11:11:AREQ&quot;, &quot;color&quot;: &quot;#ffaaff&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 2&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;KLYS:LI11:21:AREQ&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;KLYS:LI11:11:PREQ&quot;, &quot;color&quot;: &quot;#ffaaff&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 2&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
         </stringlist>
        </property>
        <property name="updateMode" stdset="0">
         <enum>PyDMTimePlot::AtFixedRate</enum>
        </property>
        <property name="timeSpan">
         <double>900.000000000000000</double>
        </property>
        <property name="autoRangeY">
         <bool>true</bool>
        </property>
        <property name="maxYRange">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="PyDMTimePlot" name="L1_blen">
        <property name="toolTip">
         <string/>
        </property>
        <property name="showXGrid">
         <bool>false</bool>
        </property>
        <property name="showYGrid">
         <bool>false</bool>
        </property>
        <property name="yAxes">
         <stringlist>
          <string>{&quot;name&quot;: &quot;Axis 1&quot;, &quot;orientation&quot;: &quot;right&quot;, &quot;label&quot;: null, &quot;minRange&quot;: -5474.9380608, &quot;maxRange&quot;: 5474.9380608, &quot;autoRange&quot;: false, &quot;logMode&quot;: false}</string>
          <string>{&quot;name&quot;: &quot;Axis 2&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: null, &quot;minRange&quot;: -5474.9380608, &quot;maxRange&quot;: 5474.9380608, &quot;autoRange&quot;: false, &quot;logMode&quot;: false}</string>
         </stringlist>
        </property>
        <property name="curves">
         <stringlist>
          <string>{&quot;channel&quot;: &quot;BLEN:LI11:359:BZ11359B_S_SUM&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;BLEN:LI11:359:BZ11359B_S_SUM&quot;, &quot;color&quot;: &quot;#ffff7f&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 1&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;PHYS:SYS1:1:F2LFB_BC11BL_TARGET&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;SIOC:SYS1:ML01:AO221&quot;, &quot;color&quot;: &quot;#ffaa00&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 1&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;KLYS:LI11:11:PREQ&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;KLYS:LI11:11:PREQ&quot;, &quot;color&quot;: &quot;#00aa7f&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 2&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;KLYS:LI11:21:PREQ&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;KLYS:LI11:21:PREQ&quot;, &quot;color&quot;: &quot;#00aa7f&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 2&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
         </stringlist>
        </property>
        <property name="updateMode" stdset="0">
         <enum>PyDMTimePlot::AtFixedRate</enum>
        </property>
        <property name="timeSpan">
         <double>900.000000000000000</double>
        </property>
        <property name="autoRangeY">
         <bool>false</bool>
        </property>
        <property name="maxYRange">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="PyDMTimePlot" name="L2_energy">
        <property name="toolTip">
         <string/>
        </property>
        <property name="showXGrid">
         <bool>false</bool>
        </property>
        <property name="showYGrid">
         <bool>false</bool>
        </property>
        <property name="yAxes">
         <stringlist>
          <string>{&quot;name&quot;: &quot;Axis 1&quot;, &quot;orientation&quot;: &quot;right&quot;, &quot;label&quot;: null, &quot;minRange&quot;: -8189.0099199999995, &quot;maxRange&quot;: 8189.0099199999995, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
          <string>{&quot;name&quot;: &quot;Axis 2&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: null, &quot;minRange&quot;: -8189.0099199999995, &quot;maxRange&quot;: 8189.0099199999995, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
         </stringlist>
        </property>
        <property name="curves">
         <stringlist>
          <string>{&quot;channel&quot;: &quot;BPMS:LI14:801:X1H&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;BPMS:LI14:801:X1H&quot;, &quot;color&quot;: &quot;#00aaff&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 1&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;LI14:KLYS:41:PDES&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;KLYS:LI14:41;PDES&quot;, &quot;color&quot;: &quot;#ffaaff&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 2&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;LI14:KLYS:61:PDES&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;KLYS:LI14:61!:PDES&quot;, &quot;color&quot;: &quot;#ffaaff&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 2&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
         </stringlist>
        </property>
        <property name="updateMode" stdset="0">
         <enum>PyDMTimePlot::AtFixedRate</enum>
        </property>
        <property name="timeSpan">
         <double>900.000000000000000</double>
        </property>
        <property name="autoRangeY">
         <bool>true</bool>
        </property>
        <property name="maxYRange">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="PyDMTimePlot" name="L2_blen">
        <property name="toolTip">
         <string/>
        </property>
        <property name="showXGrid">
         <bool>false</bool>
        </property>
        <property name="showYGrid">
         <bool>false</bool>
        </property>
        <property name="yAxes">
         <stringlist>
          <string>{&quot;name&quot;: &quot;Axis 1&quot;, &quot;orientation&quot;: &quot;right&quot;, &quot;label&quot;: null, &quot;minRange&quot;: -9579.983352836914, &quot;maxRange&quot;: 9579.983352836914, &quot;autoRange&quot;: false, &quot;logMode&quot;: false}</string>
          <string>{&quot;name&quot;: &quot;Axis 2&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: null, &quot;minRange&quot;: -33.74592, &quot;maxRange&quot;: 33.74592, &quot;autoRange&quot;: false, &quot;logMode&quot;: false}</string>
         </stringlist>
        </property>
        <property name="curves">
         <stringlist>
          <string>{&quot;channel&quot;: &quot;BLEN:LI14:888:BZ14888B_S_SUM&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;BLEN:LI14:888:BZ14888B_S_SUM&quot;, &quot;color&quot;: &quot;#ffff7f&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 1&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;PHYS:SYS1:1:F2LFB_BC14BL_TARGET&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;SIOC:SYS1:ML00:CALC144&quot;, &quot;color&quot;: &quot;#ffaa00&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 1&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;LI12:SBST:1:PHAS&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;LI12:SBST:1:PHAS&quot;, &quot;color&quot;: &quot;#00aa7f&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 2&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;LI13:SBST:1:PHAS&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;LI13:SBST:PHAS&quot;, &quot;color&quot;: &quot;#00aa7f&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 2&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;LI14:SBST:1:PHAS&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;LI13:SBST:PHAS&quot;, &quot;color&quot;: &quot;#00aa7f&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 2&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
         </stringlist>
        </property>
        <property name="updateMode" stdset="0">
         <enum>PyDMTimePlot::AtFixedRate</enum>
        </property>
        <property name="timeSpan">
         <double>900.000000000000000</double>
        </property>
        <property name="autoRangeY">
         <bool>false</bool>
        </property>
        <property name="maxYRange">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="PyDMTimePlot" name="L3_energy">
        <property name="toolTip">
         <string/>
        </property>
        <property name="showXGrid">
         <bool>false</bool>
        </property>
        <property name="showYGrid">
         <bool>false</bool>
        </property>
        <property name="yAxes">
         <stringlist>
          <string>{&quot;name&quot;: &quot;Axis 1&quot;, &quot;orientation&quot;: &quot;right&quot;, &quot;label&quot;: null, &quot;minRange&quot;: -182.49793536, &quot;maxRange&quot;: 182.49793536, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
          <string>{&quot;name&quot;: &quot;Axis 2&quot;, &quot;orientation&quot;: &quot;left&quot;, &quot;label&quot;: null, &quot;minRange&quot;: -182.49793536, &quot;maxRange&quot;: 182.49793536, &quot;autoRange&quot;: true, &quot;logMode&quot;: false}</string>
         </stringlist>
        </property>
        <property name="curves">
         <stringlist>
          <string>{&quot;channel&quot;: &quot;BPMS:LI20:2445:X1H&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;BPMS:LI20:2445:X1H&quot;, &quot;color&quot;: &quot;#00aaff&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 1&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;LI19:KLYS:41:PDES&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;KLYS:LI19:41:PDES&quot;, &quot;color&quot;: &quot;#ffaaff&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 2&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
          <string>{&quot;channel&quot;: &quot;LI19:KLYS:51:PDES&quot;, &quot;plot_style&quot;: null, &quot;name&quot;: &quot;KLYS:LI19:61:PDES&quot;, &quot;color&quot;: &quot;#ffaaff&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 1, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10, &quot;yAxisName&quot;: &quot;Axis 2&quot;, &quot;barWidth&quot;: null, &quot;upperThreshold&quot;: null, &quot;lowerThreshold&quot;: null, &quot;thresholdColor&quot;: &quot;white&quot;}</string>
         </stringlist>
        </property>
        <property name="updateMode" stdset="0">
         <enum>PyDMTimePlot::AtFixedRate</enum>
        </property>
        <property name="timeSpan">
         <double>900.000000000000000</double>
        </property>
        <property name="autoRangeY">
         <bool>true</bool>
        </property>
        <property name="minYRange">
         <double>0.000000000000000</double>
        </property>
        <property name="maxYRange">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMTimePlot</class>
   <extends>QGraphicsView</extends>
   <header>pydm.widgets.timeplot</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
